lib/vyasa_web/components/control_panel.ex

```diff
@@ -27,6 +27,7 @@ defmodule VyasaWeb.ControlPanel do
       <!-- SVG Icon Button -->
       <.control_panel_mode_indicator mode={@mode} myself={@myself} session_active?={@session.name} />
       <div
+        :if={@show_control_panel?}
```

Comment:
how the bug would present itself 

![image](https://github.com/user-attachments/assets/62967152-d0d3-4721-9f88-d2be4515257e)

see [this commit message](https://github.com/ve1ld/vyasa/pull/136/commits/998bee6e55bf502745cd8f194105a5b46370698d) for more info